### PR TITLE
Improve Clarity of Wallet Store Decoding Error Message

### PR DESCRIPTION
**Pull request by nodersteam**
## Describe your changes
modified https://github.com/nodersteam/namada/blob/456486fe61c2f5c3c60136963018224280b22f61/apps/src/lib/wallet/mod.rs#L245 to return a `Result` so the error can be propagated to the caller, and thus `eprinted` well.
## Indicate on which release or other PRs this topic is based on
v0.29.0
Closes https://github.com/anoma/namada/issues/2151
## Checklist before merging to `draft`
- [x] I have added a changelog
- [X] Git history is in acceptable state

### CODE REVIEW

**Comment by tzemanovic** [comment]
hi, thanks for the PR, but it looks like the description no longer matches that changes. Can you pls update it?

**Comment by nodersteam** [comment]
> hi, thanks for the PR, but it looks like the description no longer matches that changes. Can you pls update it?

HI! Yep, sure
I'll fix all and resolve and ping you asap

File: apps/src/lib/cli.rs [hunk 1]
```diff
@@ -2981,8 +2996,8 @@ pub mod args {
     pub const IBC_TRANSFER_MEMO_PATH: ArgOpt<PathBuf> = arg_opt("memo-path");
     pub const INPUT_OPT: ArgOpt<PathBuf> = arg_opt("input");
     pub const LEDGER_ADDRESS_ABOUT: &str =
-        "Address of a ledger node as \"{scheme}://{host}:{port}\". If the \
-         scheme is not supplied, it is assumed to be TCP.";
+        "Address of a ledger node as \"{scheme}://{host}:{port}\". If \
+         the\nscheme is not supplied, it is assumed to be TCP.";
```
Comment (karbyshev) [inline apps/src/lib/cli.rs:3000]: Is it a manual formatting? Why?

File: apps/src/lib/cli.rs [hunk 2]
```diff
@@ -6230,18 +6459,24 @@ pub mod args {
...
+            )
+            .arg(
+                HD_WALLET_DERIVATION_PATH.def().help(
+                    "HD key derivation path. Use keyword `default` to refer \
+                     to a scheme default path:\n- m/44'/60'/0'/0/0 for \
+                     secp256k1 scheme\n- m/44'/877'/0'/0'/0' for ed25519 \
+                     scheme.\nFor ed25519, all path indices will be promoted \
+                     to hardened indexes. If none is specified, the scheme \
+                     default path is used."
+                        .wrap_lines(),
```
Comment (karbyshev) [inline apps/src/lib/cli.rs:6477]: This will apparently break the help message formatting. Also, in other places.

**Review by karbyshev — COMMENTED** [review]
To be honest, I don't understand what this PR is trying to achieve and how is it related to #2151. It will also break the formatting of some help messages.

**Comment by cwgoes** [comment]
Closing this since we haven't seen any response to the feedback, but feel free to reopen in the future.

**Pull request closed without merging**